rtl/eclrtl/eclrtl.cpp

```diff
@@ -357,19 +357,19 @@ bool vunicodeNeedsNormalize(UChar * in, UErrorCode * err)
 
 void unicodeReplaceNormalized(unsigned inlen, UChar * in, UErrorCode * err)
 {
-    UChar * buff = (UChar *)rtlMalloc(inlen*2);
+    UChar * buff = (UChar *)rtlMalloc(inlen*sizeof(UChar));
     unsigned len = unorm_normalize(in, inlen, UNORM_NFC, 0, buff, inlen, err);
     while(len<inlen) buff[len++] = 0x0020;
-    memcpy(in, buff, inlen);
+    memcpy(in, buff, inlen * sizeof(UChar));
```

Comment:
Spotted when moving an equivalent function into unicodelib.  I cannot think of a way of testing this example, since it is only used in very limited situations.  Please double check.